content/docs/3_reference/3_panel/3_fields/0_pages/reference-article.txt

```diff
@@ -104,6 +104,25 @@ The default (preview) image is the first image in the folder. You can configure
 image: page.image.findBy("name", "cover")
 ```
 
+### Preview image from page files field
+
+You can use the image of each subpage `files` field as preview image:
+
+```yaml
+image: page.my_files_field.toFile
```

Comment:
We don't use snake_case anywhere across the documentation. Could you please change to camelCase.

Reply:
Done ✅

Reply:
Thanks!